neard/res/mainnet_genesis.json

```diff
@@ -197,39 +212,87 @@
     {
       "account_id": "nfvalidator1.near",
       "public_key": "ed25519:14pWWRutZtGFKX4B8q89KVFaUWY1Cqu1JcqYXhCDeFh1",
-      "amount": "50000000011865419790795122041"
+      "amount": "50000000000000000000000000000"
     },
     {
       "account_id": "nfvalidator2.near",
       "public_key": "ed25519:BwZk4bkYJxo79P2vSRw2uk1nfiqEfVkHvr5p8eVsqASC",
-      "amount": "50000000011875276807113550000"
+      "amount": "50000000000000000000000000000"
     },
     {
       "account_id": "nfvalidator3.near",
       "public_key": "ed25519:DMz11tmPvhdqpi7CzP2JULeeSE8SxYRD8pys5nKke4FS",
-      "amount": "50000000011874181583078169115"
+      "amount": "50000000000000000000000000000"
     },
     {
       "account_id": "nfvalidator4.near",
       "public_key": "ed25519:Fi3CQDHJoviKazVR27YmfFzWcFnvmoPBKEDd9ouq5Tjx",
-      "amount": "50000000011875276807113550000"
+      "amount": "50000000000000000000000000000"
     }
   ],
   "transaction_validity_period": 86400,
+  "protocol_reward_rate": [
+    0,
+    1
+  ],
   "max_inflation_rate": [
-    1,
-    20
+    0,
```

Comment:
Is this correct?

Reply:
yes we inflation is disabled before phase 2